Tests/TestingTests/MiscellaneousTests.swift

```diff
@@ -123,6 +123,15 @@ struct SendableTests: Sendable {
 
   @Test(.hidden, arguments: FixtureData.stringReturningClosureArray)
   func parameterizedAcceptingFunction(f: @Sendable () -> String) {}
+
+  @Test(.hidden) func throwKeywordInExpectation() throws {
```

Comment:
These are test cases that would have failed compile prior to this fix?

Reply:
Yep.